app/services/status_message_creation_service.rb

```diff
@@ -19,20 +19,9 @@ def create(params)
 
   def build_status_message(params)
     public = params[:public] || false
-    filter_mentions params
```

Comment:
you should still filter on send, because otherwise old pods will display the post to people who shouldn't see it.